RELEASE_NOTES.md

```diff
@@ -109,7 +109,7 @@ making a backup is strongly advised.
 
 ## New Features And Changes
 
-- Add support for curved gradients. This can be helpful when putting a gradient mask on an  image with a horizon line that is curved because lens distortion. This can also be used towards artistic goals.
+- Add support for curved gradients. This can be helpful when putting a gradient mask on an image with a horizon line that is curved due to lens distortion. This can also be for towards artistic goals.
```

Comment:
please can you remove the word "towards"

Reply:
yes, done.